sql/_05_plcsql/_01_testspec/_03_statement/_13_loop/_06_for_dynamic_sql/cases/normal_basic_4.sql

```diff
@@ -0,0 +1,25 @@
+--+ server-message on
+-- Verification for CBRD-24708
+
+create or replace function hostvar_test(i_id int, i_name varchar) return varchar as
+    s_id int := i_id;
+    s_name varchar(10) := i_name;
+
+    id int;
+    name varchar;
+begin
+    id := 0;
+    for r in (select charset_id, charset_name from db_charset where charset_id = s_id and charset_name = s_name) loop
+        id := r.charset_id;
+        name := r.charset_name;
+    end loop;
+
+    return id || ', ' || name;
+end;
+
+select hostvar_test(5,'utf8') from dual;
+select hostvar_test(7,'utf-8') from dual; -- return NULL, because data is not found
+
```

Comment:
-- return NULL, because data is not found
select hostvar_test(7,'utf-8') from dual;

Reply:
reflected review